### PR TITLE
Documentation: teaching: fix typo

### PR DESCRIPTION
**Pull request by etzl**
Signed-off-by: Erfan Zamani <erfanzamani3445@gmail.com>

### CODE REVIEW

File: Documentation/teaching/lectures/interrupts.rst [hunk 1]
```diff
@@ -842,24 +842,24 @@ low latency requirements and high frequency:
 
       /* PLEASE, avoid to allocate new softirqs, if you need not _really_ high
          frequency threaded job scheduling. For almost all the purposes
-	 tasklets are more than enough. F.e. all serial device BHs et
```
Comment (etzl) [inline Documentation/teaching/lectures/interrupts.rst:845]: My editor must be changing these, is that a problem?

Comment (dbaluta) [reply Documentation/teaching/lectures/interrupts.rst:845]: yes. we should not modify random lines by adding/removing spaces.

Comment (etzl) [reply Documentation/teaching/lectures/interrupts.rst:845]: can I make another commit on top? or should I edit my previous commits?

Comment (etzl) [reply Documentation/teaching/lectures/interrupts.rst:845]: No, I meant locally. But I see what you mean, I should edit my last commit using `--amend`

**Comment by etzl** [comment]
Ok, I've fixed the line indentations. I reverted my last commit and changed the file. But I realized that I could revert to my first commit and prevent the changes from being committed. So let me know

**Comment by dbaluta** [comment]
@etzl looks good now. Only concern is a merge commit which shouldn't be there. Can you please remove it and then force push.

image

**Review by dbaluta — APPROVED** [review]

**Comment by dbaluta** [comment]
@etzl merged. thanks!

**Comment by etzl** [comment]
@dbaluta Glad I could help

**Review by tavip — COMMENTED** [review]
LGTM, thank you @etzl !

**Pull request merged**